DEVELOPMENT.md

```diff
@@ -2,6 +2,11 @@
 
 TODO: Complete me! Should be of similar quality to the ECM documentation.
 
+## Contributing
+
+For compliance reasons, all pull requests **must** be submitted with a Jira ID in the pull request title.
+Include the Jira ID at the beginning of the title, enclosed in square brackets.
```

Comment:
small nit: maybe we can remove the requirement of "enclosed in square brackets". I don't think that's mandatory no?

Reply:
Yeah I never put them in square brackets 😆

Reply:
I'd also say, if there is more than one relevant ticket, include them all, eg `WM-1997, WM-2002, WM-2005: Fix for many bugs with the same root cause`, etc.